test/kitchen/test-definitions/platforms-common.yml

```diff
@@ -5,7 +5,7 @@
   aptrepo = "[signed-by=/usr/share/keyrings/datadog-archive-keyring.gpg] http://apttesting.datad0g.com/"
   aptrepo_dist = "pipeline-#{ENV['DD_PIPELINE_ID']}"
   yumrepo = "http://yumtesting.datad0g.com/testing/pipeline-#{ENV['DD_PIPELINE_ID']}/#{ENV['MAJOR_VERSION']}/x86_64/"
-  yumrepo_suse = "http://yumtesting.datad0g.com/suse/testing/pipeline-#{ENV['DD_PIPELINE_ID']}/#{ENV['MAJOR_VERSION']}/x86_64/"
+  yumrepo_suse = "http://yumtesting.datad0g.com/suse/testing/pipeline-#{ENV['DD_PIPELINE_ID']}/#{ENV['MAJOR_VERSION']}/#{ENV['SUSE_REPO_ARCH']}/"
```

Comment:
There is already a `KITCHEN_ARCH` variable, set to `x86_64` or `arm64`, why not reuse it instead of defining a new variable?

Reply:
`aarch64` needs to be used here, and it felt cleaner to have a dedicated variable with some limited reach than to implement a substitution here. It's definitely doable though

Reply:
Having two variables that convey the same information + can be set independently and in contradictory ways - someone could mistakenly set `KITCHEN_ARCH=x86_64` and `SUSE_REPO_ARCH=aarch64` in a job - doesn't feel cleaner to me.

Having arch mappings in tooling is pretty common (see [one example in our own code](https://github.com/DataDog/datadog-agent/blob/c8c20c1b1d6dc39be0e47f5f6440d14af77c8f3e/tasks/system_probe.py#L47-L55)) due to different systems naming similar architectures with different names, so I don't think it's a big burden.

Reply:
Fair enough :) I added a mapping function in `platforms-common.yml`